### PR TITLE
MarkCompleted button integrated

### PR DESCRIPTION
**Pull request by diegotc86**
Now we can do all the process:
login->pick restaurant->select dishes->create order->complete order

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ function addNewOrder(order) {
 }
 
 function markCompleted(id) {
+  console.log("llego a actions");
```
Comment (luciagirasoles) [inline src/actions.js]: don't forget to remove this in the future

File: src/reducers/reducer.js [hunk 2]
```diff
@@ -52,6 +52,7 @@ function appReducer(state = initialState, action = {}) {
       };
     }
     case "MARK_COMPLETED": {
+      console.log("llego a reducer");
```
Comment (luciagirasoles) [inline src/reducers/reducer.js]: even here

File: src/views/order-map-view.js [hunk 3]
```diff
@@ -21,26 +23,27 @@ function OrderView(props) {
...
   const restaurant = props.restaurants[currentOrder.restaurant_id];
-  console.log(restaurant);
 
   const restaurantGeo = [
     +restaurant.address.latitud,
     +restaurant.address.longitud
   ];
 
+  function handleComplete() {
+    props.markCompleted(+props.id);
```
Comment (MariethPerez) [inline src/views/order-map-view.js]: Hi, please Could explain me How works (+propo.id)?

File: src/actions.js (hunk 1 above)
Comment (diegotc86) [reply src/actions.js]: :+1:

File: src/reducers/reducer.js (hunk 2 above)
Comment (diegotc86) [reply src/reducers/reducer.js]: :+1:

File: src/views/order-map-view.js (hunk 3 above)
Comment (diegotc86) [reply src/views/order-map-view.js]: Hi Marieth, the plus symbol "+" ensure that props.id is converted to a number

**Pull request merged**